src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala

```diff
@@ -60,7 +60,8 @@ object LambdaResponse extends Base64Utils {
 case class LambdaResponse(
   statusCode: Int,
   maybeBody: Option[String],
-  headers: Map[String, String] = Map("Content-Type" -> "application/json"))
+  headers: Map[String, String] = Map("Content-Type" -> "application/json; charset=UTF-8", "cache-control" -> "max-age=0")
```

Comment:
HTTP headers are supposed to be case insensitive, so it should not matter.
But, sometimes clients/servers get it wrong: is cache-control all lower case okay on both IOS and Android?

Reply:
The current service uses this form, so I think it's safe to assume this is fine for all clients